e2e/node/e2e.test.ts

```diff
@@ -944,9 +948,7 @@ describe(`End-to-end access grant tests for environment [${environment}}]`, () =
     });
   });
 
-  // This is skipped until ESS supports recursive Access Grants
-  // eslint-disable-next-line jest/no-disabled-tests
-  describe.skip("access grants can be recursive or only apply to the target resource", () => {
+  describe("access grants can be recursive or only apply to the target resource", () => {
```

Comment:
Note on the two disabled tests above, those can probably use inherit / recursive access grants to function as expected

Reply:
I rephrased the describe, because I think it makes sense as a single block, and that's a good point, I have re-enabled them: 34f00a0d742b846075b69b360b5b6d25dd14a3ea